service/src/test/java/uk/gov/hmcts/reform/fpl/service/PlacementServiceTest.java

```diff
@@ -192,6 +198,69 @@ void shouldUpdateExistingPlacement() {
         }
     }
 
+    @Nested
+    class GetBinaryUrlsForOrderAndNotices {
+
+        @Test
+        void shouldReturnEmptyListWhenNoPlacementsExist() {
+            assertThat(placementService.getBinaryUrlsForOrderAndNotices(emptyList(), PLACEMENT_ORDER)).isEmpty();
+        }
+
+        @Test
+        void shouldReturnEmptyListWhenPlacementWithNoOrderAndNotices() {
+            List<Element<Placement>> placements = wrapElements(placementWithoutOrderAndNotices());
+
+            assertThat(placementService.getBinaryUrlsForOrderAndNotices(placements, PLACEMENT_ORDER)).isEmpty();
+        }
+
+        @Test
+        void shouldReturnListOfIdsForSpecifiedTypeWhenPlacementsIsPopulated() {
+            String binaryUrl = "example binary url link";
+            PlacementOrderAndNotices.PlacementOrderAndNoticesType type = PLACEMENT_ORDER;
+
+            List<Element<Placement>> placements = wrapElements(
+                placement(binaryUrl, type),
+                placement("other url", OTHER));
+
+            assertThat(placementService.getBinaryUrlsForOrderAndNotices(placements, type)).containsOnly(binaryUrl);
+        }
+
+        @Test
+        void shouldReturnEmptyListWhenBinaryUrlIsNotPresent() {
+            PlacementOrderAndNotices.PlacementOrderAndNoticesType type = PLACEMENT_ORDER;
+
+            List<Element<Placement>> placements = wrapElements(
+                Placement.builder()
+                    .orderAndNotices(wrapElements(PlacementOrderAndNotices.builder()
+                        .type(type)
+                        .build()))
+                    .build());
```

Comment:
[not just `placement(...)`?](https://github.com/hmcts/fpl-ccd-configuration/pull/863)